docssrc/src/permissions.md

```diff
@@ -117,3 +117,82 @@ Now we declare our command with arguments. We use a `PlayerArgument` and apply t
 > As you can see, there are multiple ways of applying permissions to commands with arguments. In the `/god` command shown above, the permission was applied to the whole command. In the `/kill` command shown above, the permission was applied to the argument.
 >
 > There's not really much difference between the two methods, but I personally would use _argument permissions_ as it has greater control over arguments.
+
+-----
+
+## Child Base Permissions
+### Example - /economy command with argument permissions
```

Comment:
Move this heading below this paragraph, and just above "For example, say we're..."

Reply:
> Move this heading below this paragraph, and just above "For example, say we're..."

Im not too sure what you mean by this but I've just pushed a commit that I think fixes it?

Reply:
Something along the lines of this:

```md
## Child-based Permissions

Child-based permissions allow you to group permissions together. We achieve this by laying out our permission groups in the `plugin.yml` file which Bukkit registers as valid permissions. When the CommandAPI checks if our player has a permission, Bukkit considers if they have the child of a permission as well. This not only keeps permissions easier to manage, it also makes your code cleaner and gives you a nice place to layout out all of your permissions, detailing what they do and what other permissions inherit them.

### Example - /economy command with argument permissions

For example, say we're registering a command `/economy`
```

Reply:
ah now that makes sense okay thanks!